README.rst

```diff
@@ -34,7 +34,7 @@ Annoy was built by `Erik Bernhardsson <http://www.erikbern.com>`__ in a couple o
 Summary of features
 -------------------
 
-* Euclidean distance or cosine distance (where cosine distance uses Euclidean distance of normalized vectors which equals 2-2*cosine similarity)
+* Euclidean distance or cosine distance, where cosine distance uses Euclidean distance of normalized vectors = sqrt(2-2*cos(u, v))
```

Comment:
no – doesn't change anything except the values returned by `get_distance`

still on the fence about merging this since it might break existing users, but I think i'll merge it into master and wait a few weeks to publish it to pypi